packages/shared/src/components/fields/MarkdownInput/SavingLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Loader } from '../../Loader';
+import VIcon from '../../icons/V';
+import { IconSize } from '../../Icon';
+
+interface SavingLabelProps {
+  isUptoDate?: boolean;
+  isUpdating?: boolean;
+  className?: string;
+}
+
+export function SavingLabel({
+  isUptoDate,
```

Comment:
I don't see this being passed anywhere to `<SavingLabel />`?

Reply:
Oh, I must have missed it. Will check this before merging.